tests/test_estimateRadialVelocity.py

```diff
@@ -0,0 +1,174 @@
+from pfs.drp.stella.tests import runTests
+from pfs.datamodel import MaskHelper
+from pfs.datamodel.observations import Observations
+from pfs.datamodel.pfsFiberArray import PfsFiberArray
+from pfs.datamodel.pfsSimpleSpectrum import PfsSimpleSpectrum
+from pfs.datamodel.target import Target
+
+import numpy as np
+import scipy.interpolate
+import scipy.signal
+
+
+class EstimateRadialVelocityTestCase(lsst.utils.tests.TestCase):
+    def setUp(self):
+        self.estimateRadialVelocity = EstimateRadialVelocityTask(config=EstimateRadialVelocityConfig())
+        # It may be fearsome to use default_rng()
+        # because numpy is planning to change the default RNG again as of 2021-09-15.
+        # We call PCG64 explicitly.
+        self.np_random = np.random.Generator(np.random.PCG64(seed=0x789abcdef0123456))
+
+    def test(self):
+        """Test getRadialVelocity()
+        """
+        refSpectrum = self.createRefSpectrum()
+        permissibleError = 1  # km/s
+
+        for snr, trueVelocity in [
+            (25, -200),
+            (50, -100),
+            (100, 0),
+            (200, 100),
+            (400, 200),
+        ]:
+            obsSpectrum = self.createObsSpectrum(refSpectrum, trueVelocity, snr)
+            velocity, velocity_err = self.estimateRadialVelocity.getRadialVelocity(obsSpectrum, refSpectrum)
+            self.assertLess(abs(velocity - trueVelocity), permissibleError + 2*velocity_err)
+
+    def createRefSpectrum(self):
+        """
+        Create a fake reference spectrum.
+
+        Returns
+        -------
+        spectrum : `pfs.datamodel.pfsSimpleSpectrum.PfsSimpleSpectrum`
+        """
+        wavelength = np.linspace(300, 1300, 100000)
+
+        peak = self.np_random.uniform(wavelength[0], wavelength[-1], size=200).reshape(-1, 1)
+        depth = self.np_random.beta(1, 5, size=peak.shape)
+        flux = np.prod(1.0 - depth * createRefLsf(wavelength.reshape(1, -1), peak), axis=0)
+
+        target = Target(0, 0, "0,0", 0)
+        mask = np.zeros(len(wavelength), dtype=int)
+        flags = MaskHelper()
+
+        return PfsSimpleSpectrum(target, wavelength, flux, mask, flags)
+
+    def createObsSpectrum(self, refSpectrum, trueVelocity, snr):
+        """
+        Create a fake observed spectrum.
+
+        Parameters
+        ----------
+        refSpectrum : `pfs.datamodel.pfsSimpleSpectrum.PfsSimpleSpectrum`
+            Reference spectrum.
+        trueVelocity : `float`
+            Radial velocity of the observed object.
+        snr : `float
+            Signal-to-noise ratio.
+
+        Returns
+        -------
+        spectrum : `pfs.datamodel.pfsFiberArray.PfsFiberArray`
+        """
+        refWavelength = refSpectrum.wavelength
+        refFlux = refSpectrum.flux
+
+        # Select any 3nm range made of an odd number of sampling points
+        index = np.nonzero((800 <= refWavelength) & (refWavelength <= 803))[0]
```

Comment:
Can you make this range configurable? It's usually good to avoid hard-coded "magic numbers".